examples/fishjam-chat/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
+    "eas-build-pre-install": "cd ../.. && yarn && yarn build",
```

Comment:
**question:** Do we need this commend? If something can call `eas-build-pre-install` can't it also switch directories? 🤔

Reply:
I don't think so. Eas calls this script by itself, I didn't see any way of selecting directory for one of those hooks. And this command is needed because eas doesn't understand that it should run `yarn` in upper directories of monorepo

Reply:
Curious, maybe we can run `yarn workspace fishjam-mobile-sdk install && yarn workspace fishjam-mobile-sdk build` instead? So we won't have to run `cd` command.